spark/components/icons/vanilla/icons.stories.js

```diff
@@ -1,3 +1,1411 @@
+    <use xlink:href="#feedback-filled-small" />
+  </svg>         
+  `
+);
+
+export const findAgent = () => (
+  `
+  <svg class="sprk-c-Icon sprk-c-Icon--xl" viewBox="0 0 100 100">
+    <use xlink:href="#find-agent" />
+  </svg>         
+  `
+);
+
+export const free = () => (
+  `
+  <svg class="sprk-c-Icon sprk-c-Icon--xl" viewBox="0 0 100 100">
+    <use xlink:href="#free" />
+  </svg>         
+  `
+);
+
+export const heat = () => (
+  `
+  <svg class="sprk-c-Icon sprk-c-Icon--xl" viewBox="0 0 100 100">
+    <use xlink:href="#heat" />
+  </svg>         
+  `
+);
+
+export const home = () => (
+  `
+  <svg class="sprk-c-Icon sprk-c-Icon--xl" viewBox="0 0 100 100">
+    <use xlink:href="#home" />
+  </svg>         
+  `
+);
+
+export const homeImprovement = () => (
+  `
+  <svg class="sprk-c-Icon sprk-c-Icon--xl" viewBox="0 0 100 100">
+    <use xlink:href="#home-improvement" />
+  </svg>         
+  `
+);
+
+export const homeShopping = () => (
+  `
+  <svg class="sprk-c-Icon sprk-c-Icon--xl" viewBox="0 0 100 100">
+    <use xlink:href="#home-shopping" />
+  </svg>         
+  `
+);
+
+export const homeSize = () => (
+  `
+  <svg class="sprk-c-Icon sprk-c-Icon--xl" viewBox="0 0 100 100">
+    <use xlink:href="#home-size" />
+  </svg>         
+  `
+);
+
+export const information = () => (
+  `
+  <svg class="sprk-c-Icon sprk-c-Icon--xl" viewBox="0 0 100 100">
+    <use xlink:href="#information" />
+  </svg>         
+  `
+);
+
+export const informationFilled = () => (
+  `
+  <svg class="sprk-c-Icon sprk-c-Icon--filled sprk-c-Icon--xl" viewBox="0 0 100 100">
+    <use xlink:href="#information-filled" />
+  </svg>         
+  `
+);
+
+export const informationFilledSmall = () => (
+  `
+  <svg class="sprk-c-Icon sprk-c-Icon--filled" viewBox="0 0 100 100">
+    <use xlink:href="#information-filled-small" />
+  </svg>         
+  `
+);
+
+export const instagram = () => (
+  `
+  <svg class="sprk-c-Icon sprk-c-Icon--xl" viewBox="0 0 100 100">
+    <use xlink:href="#instagram" />
+  </svg>         
+  `
+);
+
+export const instagramTwoColor = () => (
+  `
+  <svg class="sprk-c-Icon sprk-c-Icon--xl" viewBox="0 0 100 100">
+    <use xlink:href="#instagram-two-color" />
+  </svg>         
+  `
+);
+
+export const land = () => (
+  `
+  <svg class="sprk-c-Icon sprk-c-Icon--xl" viewBox="0 0 100 100">
+    <use xlink:href="#land" />
+  </svg>         
+  `
+);
+
+export const landline = () => (
+  `
+  <svg class="sprk-c-Icon sprk-c-Icon--xl" viewBox="0 0 100 100">
+    <use xlink:href="#landline" />
+  </svg>         
+  `
+);
+
+export const landlineFilled = () => (
+  `
+  <svg class="sprk-c-Icon sprk-c-Icon--filled sprk-c-Icon--xl" viewBox="0 0 100 100">
+    <use xlink:href="#landline-filled" />
+  </svg>         
+  `
+);
+
+export const landlineFilledSmall = () => (
+  `
+  <svg class="sprk-c-Icon sprk-c-Icon--filled" viewBox="0 0 100 100">
+    <use xlink:href="#landline-filled" />
```

Comment:
```suggestion
    <use xlink:href="#landline-filled-small" />
```
This one's supposed to be landline-filled-small